learn/configuration/instance_options.md

```diff
@@ -44,8 +44,84 @@ Environment variables for command-line flags accept `n`, `no`, `f`, `false`, `of
 
 Environment variables are always identical to the corresponding command-line option, but prepended with `MEILI_` and written in all uppercase. **Some options (e.g., `--import-snapshots`) are not available as environment variables.**
 
+## Configuration file
+
+Meilisearch accepts a configuration file in the `.toml` format as an alternative to command-line options and environment variables. Configuration files can be easily shared and versioned, and allow you to define multiple options.
+
+**When used simultaneously, environment variables overwrite the configuration file, and command-line options overwrite environment variables.**
+
+You can download the configuration file using the following command:
+
+<CodeSamples id="instance_options_config_file_download_1" />
+
+By default, Meilisearch will use `config.toml` in the working directory as the default configuration file. You will see it when you launch Meilisearch:
+
+```
+888b     d888          d8b 888 d8b                                            888
+8888b   d8888          Y8P 888 Y8P                                            888
+88888b.d88888              888                                                888
+888Y88888P888  .d88b.  888 888 888 .d8888b   .d88b.   8888b.  888d888 .d8888b 88888b.
+888 Y888P 888 d8P  Y8b 888 888 888 88K      d8P  Y8b     "88b 888P"  d88P"    888 "88b
+888  Y8P  888 88888888 888 888 888 "Y8888b. 88888888 .d888888 888    888      888  888
+888   "   888 Y8b.     888 888 888      X88 Y8b.     888  888 888    Y88b.    888  888
+888       888  "Y8888  888 888 888  88888P'  "Y8888  "Y888888 888     "Y8888P 888  888
+
+Config file path:       "./config.toml"
+```
+
+You can override the default configuration file using the `MEILI_CONFIG_FILE_PATH` environment variable or the `--config-file-path` CLI option:
+
+:::: tabs
+::: tab CLI
+
+```sh
+./meilisearch --config-file-path="./config.toml"
+```
+
+:::
+
+::: tab Environment variable
+
+Linux/MacOS:
+
+```sh
+export MEILI_CONFIG_FILE_PATH="./config.toml"
+./meilisearch
+```
+
+Windows:
```

Comment:
Just by curiosity? Why do we now explain how to set an environment variable with Windows since we don't do it [here](https://docs.meilisearch.com/learn/configuration/instance_options.html#environment-variables)?

Reply:
Good catch. I'll update it here too